### PR TITLE
Use Glean Dictionary to fetch Fenix metadata

### PR DESCRIPTION
**Pull request by wlach**
At present, this only includes better updates on status, but there's
lots of room to extend this further.

### CODE REVIEW

**Comment by wlach** [comment]
This is related to #1581, though stops short of implementing everything described in that issue.

File: rollup.config.js
Comment (wlach) [inline rollup.config.js:38]: We should set this to https://dictionary.telemetry.mozilla.org before landing. Need mozilla/glean-dictionary#882 and a new release

**Comment by wlach** [comment]
This fixes some cases where we were incorrectly reporting a metric as active. Less important but still desirable is that we now display the metrics correctly (before we replaced the category/name seperator with an underscore). For example...

Before:

image

After:

image

https://glam.telemetry.mozilla.org/fenix/probe/avif_dav1d_decode_error/explore?aggType=avg

File: src/state/api.js
Comment (Iinh) [inline src/state/api.js:127]: Why is it `json[0]` if firefox desktop?

File: src/routing/wrappers/Layout.svelte
Comment (Iinh) [inline src/routing/wrappers/Layout.svelte:21]: I'm wondering if using the new metric name format (e.g. old: `avif_dav1d_decode_error` vs new: `avif.dav1d_decode_error`) will affect the `View SQL Query` feature, since it seems like we're using the old format for our queries? 

What do you think @robhudson, if you're available?

**Review by Iinh — APPROVED** [review]
This looks really great! Thank you for your work @wlach.

File: src/state/api.js
Comment (wlach) [reply src/state/api.js:127]: See the comment above on lines 124-125: the probe search service returns an array (with a single result)

Comment (Iinh) [reply src/state/api.js:127]: oops, sorry I missed that!

File: src/routing/wrappers/Layout.svelte
Comment (wlach) [reply src/routing/wrappers/Layout.svelte:21]: Good point, we need to rewrite this probe name. Easily done.

**Comment by wlach** [comment]
Thanks @Iinh! Let's wait until Monday to merge this.

**Comment by wlach** [comment]
I did up a quick new release of the Glean Dictionary with CORS enabled on `/data/`:

https://github.com/mozilla/glean-dictionary/releases/tag/v1.14.0

So we can land this now.

**Pull request merged**
